mage_ai/data_cleaner/transformer_actions/base.py

```diff
@@ -126,7 +127,10 @@ def execute(self, df, **kwargs):
 
         if self.action_type == ActionType.FILTER:
             return df.loc[df_output.index][df_output.columns]
-        elif self.action_type == ActionType.ADD:
+        elif self.action_type == ActionType.ADD and not (
+            len(self.action['action_arguments']) == 0
```

Comment:
what's this condition for? this branch is to select columns from the df_output. it will error when action['outputs'] is empty or action['outputs'] don't exist in df_output.

Reply:
Previously, if no UDF was specified and no action arguments were specified with `add`, then the output columns would not get added to`df_output`, causing a KeyError. To resolve this case, a check was added to see if either (a) no columns were specified for the transformation or (b) no UDF was specified for the transformation. Since the 'add' template is currently not needed, I can revert this change.